pehlaschoollibrary/proj.android-studio/app/src/com/maq/pehlaschool/library/SelectActivity.java

```diff
@@ -232,11 +232,23 @@ public static void updateStringLocale( Context context, String locale)
             case "bengali":
```

Comment:
Add todo for the switch statement which has to be removed after the intent call update

Reply:
Added TODO for the switch statement.